core/server/common/src/main/java/alluxio/RpcUtils.java

```diff
@@ -187,6 +193,10 @@ private static String getQualifiedFailureMetricName(String methodName) {
     return getQualifiedMetricNameInternal(methodName + "Failures");
   }
 
+  private static String getQualifiedInProgressMetricName(String methodName) {
```

Comment:
Can you update `getQualifiedMetricNameInternal` to accept `String...components` to attach to the name instead?

Reply:
done